compatimetrics.py

```diff
@@ -0,0 +1,501 @@
+import pandas as pd
+import numpy as np
+from sklearn.metrics import accuracy_score, mean_squared_error as mse
+
+
+def mean_squared_difference(pred1, pred2):
+    """ Calculates mean squared difference of two prediction vectors, which is
+    measure of similarity between two regression models prediction.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+
+    Returns
+    -------
+    Numeric value
+        MSD value of given prediction pair
+    """
+    return mse(pred1, pred2)
+
+
+def root_mean_squared_difference(pred1, pred2):
+    """ Calculates root of mean squared difference of two prediction vectors,
+    which is measure of similarity between two regression models predictions.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+
+    Returns
+    -------
+    Numeric value
+        RMSD value of given predictions pair
```

Comment:
As above it is better to avoid such functions and mse function has _squared_ arguments which makes the calculation of rmse